specification/common-types/resource-management/v4/privatelinks.json

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
Why add common type v4?

Reply:
A new property was added to a definition which was flagged as a breaking change for any services using that definition. That forced a new version of the doc.